src/index.ts

```diff
@@ -2458,9 +2458,9 @@ export enum ConsentSource {
   AuditLogDefault = 'auditLog.default', // Overriden to defaults in audit log
   AuditLogManual = 'auditLog.manual', // Overrode one purpose in audit log
 
-  // Ketch consent all purposes opt-in/out
-  KetchConsentOptIn = 'ketch.consentOptIn',
-  KetchConsentOptOut = 'ketch.consentOptOut',
+  // Ketch accept/reject all purposes
+  KetchAcceptAllConsent = 'ketch.acceptAllConsent',
+  KetchRejectAllConsent = 'ketch.rejectAllConsent',
```

Comment:
```suggestion
  KetchRejectAllConsent = 'router.rejectAllConsent',
```

Reply:
To match the existing consent source:   `RouterSetConsent = 'router.setConsent'`